Makefile

```diff
@@ -151,12 +158,31 @@ clean:
 # Freedom Studio dev team.
 #############################################################
 
+#############################################################
+# Prints api version message.  This version should be 
+# incremented if the list-targets, list-target-tags.
+# list-programs, list-options, or standalone targets change
+# in any way that affectz Freedom Studio requirements.
+# A given version of Freedom Stddio may not support 
+# 'future' versions of freedom-e-sdk.  Backward support
+# should be supported.
+#############################################################
+.PHONY: api-version
+api-version:
+	@echo 1.0
```

Comment:
What API is this versioning? Can we make it more specific?

Reply:
This, again, would require careful manual management.  I'm just worried that, at some point, we might make changes to the recipes (or add new recipes ) that Freedom Studio relies on and unless I have some way of knowing this, Freedom Studio could be rendered quite broken if a user points to a freedom-e-sdk instance that is incompatible, or older, or newer.  FS wont be able to accurately detect or report this to the user.  Things will just break.  Having some kind of 'api version' will at least let me maintain backward compatibility, and detect when the user points to a sdk that is too new for the given version of FS.  I'm completely open to alternate solutions.  This PR is primarily just get the conversation started. :)

Reply:
My issue is primarily with the name `api-version`. Can we call it `freedomstudio-api-version`?

Reply:
Absolutely.  That's a much better name.